pkg/ls/fileinfo_plan9.go

```diff
@@ -39,15 +40,7 @@ func FromOSFileInfo(path string, fi os.FileInfo) FileInfo {
 		Name: fi.Name(),
 		Mode: fi.Mode(),
 		// Plan 9 UIDs from the file system are strings.
-		// os.FileInfo only allows ints.
```

Comment:
I think this one change and the flag change should be separate PRs, which will simplify bisecting should it ever be needed.